storage/google/cloud/storage/client.py

```diff
@@ -194,7 +194,7 @@ def lookup_bucket(self, bucket_name):
         except NotFound:
             return None
 
-    def create_bucket(self, bucket_name):
+    def create_bucket(self, bucket_name, requester_pays=None):
```

Comment:
Note that I think the `user_project` flag (added in #3490) is right for setting in `Client.bucket()` and `Bucket.__init__()`, because it is entirely about how the new client-side bucket instance is used (rather than the "real" server-side bucket).